tests/cells/cell_plugin.test.ts

```diff
@@ -69,6 +69,62 @@ describe("getCellText", () => {
     expect(result).toBeCancelledBecause(CommandResult.TargetOutOfSheet);
   });
 
+  test("update cell without any modification", () => {
+    const model = new Model();
+    const sheetId = model.getters.getActiveSheetId();
+    const result = model.dispatch("UPDATE_CELL", {
+      sheetId,
+      col: 0,
+      row: 0,
+    });
+    expect(result).toBeCancelledBecause(CommandResult.NoChanges);
+  });
+
+  test("update cell with only the same content as before", () => {
+    const model = new Model();
+    const sheetId = model.getters.getActiveSheetId();
+    setCellContent(model, "A1", "hello");
+    setCellFormat(model, "A1", "#,##0.0");
+    setStyle(model, "A1", { bold: true });
+    const result = model.dispatch("UPDATE_CELL", {
+      sheetId,
+      col: 0,
+      row: 0,
+      content: "hello",
+    });
+    expect(result).toBeCancelledBecause(CommandResult.NoChanges);
+  });
+
+  test("update cell with only the same format as before", () => {
+    const model = new Model();
+    const sheetId = model.getters.getActiveSheetId();
+    setCellContent(model, "A1", "0");
+    setCellFormat(model, "A1", "#,##0.0");
+    setStyle(model, "A1", { bold: true });
+    const result = model.dispatch("UPDATE_CELL", {
+      sheetId,
+      col: 0,
+      row: 0,
+      format: "#,##0.0",
+    });
+    expect(result).toBeCancelledBecause(CommandResult.NoChanges);
+  });
+
+  test("update cell with only the same style as before", () => {
```

Comment:
maybe add a test with all 3 content, style and border.
Also add a test for out of sheet.

Reply:
I meant format, not border